onnxruntime/core/providers/cpu/ml/svmclassifier.cc

```diff
@@ -99,24 +117,34 @@ Status SVMClassifier<T>::Compute(OpKernelContext* ctx) const {
   {
     int64_t current_weight_0 = n * stride;
     int64_t maxclass = -1;
-    double maxweight = 0.f;
     std::vector<float> decisions;
     std::vector<float> scores;
     std::vector<float> kernels;
     std::vector<int64_t> votes;
+    float sum;
+
+    if (vector_count_ == 0 && mode_ == SVM_TYPE::SVM_LINEAR) {
+      // This was in the original code but it does not appear in libsvm or scikit-learn.
+      for (int64_t j = 0; j < class_count_; j++) {  //for each class
```

Comment:
What do you mean? Is the computation in the `kernel_dot` below not included in LibSVM?

Reply:
Comment removed. There is no support vector, this code is just a dot product similar to a logistic regression.